package.json

```diff
@@ -10,13 +10,14 @@
     "test": "jest"
   },
   "dependencies": {
-    "@magicbell/react-headless": "4.5.5",
+    "@magicbell/react-headless": "https://pkg.pr.new/@magicbell/react-headless@391.tgz",
     "@react-native-async-storage/async-storage": "^2.0.0",
     "@react-navigation/native": "^6.1.14",
     "@react-navigation/native-stack": "^6.9.22",
     "expo": "^51.0.0",
     "expo-application": "~5.9.1",
     "expo-notifications": "~0.28.3",
+    "magicbell": "^4.1.0",
```

Comment:
In #11 I went ahead and removed this explicit dependency as `@magibell/react-headless` carries it already.

Reply:
Even better :)